module/observationSubmissions/helper.js

```diff
@@ -356,6 +418,60 @@ module.exports = class ObservationSubmissionsHelper {
         })
     }
 
+
+    /**
+    * List observation submissions
+    * @method
+    * @name list
+    * @param {String} - entityId
+    * @param {String} - solutionId
+    * @param {String} - observationId
+    * @returns {Object} - list of submissions
+    */
+
+    static list(entityId, solutionId, observationId) {
+        return new Promise(async (resolve, reject) => {
+
+            try {
+
+                let queryObject = {
+                    entityId: entityId,
+                    solutionId: solutionId,
+                    observationId: observationId
+                }
+
+                let projection = [
+                    "status",
+                    "submissionNumber",
+                    "entityId",
+                    "entityExternalId",
+                    "entityType",
+                    "createdAt",
+                    "updatedAt"
+                ]
+
+                let result = await this.observationSubmissionsDocument
+                    (
+                     queryObject,
+                     projection,
+                     "createdAt"
```

Comment:
@deepakr123  Does this createdAt works ??

Reply:
yes it works